frontend/src/metabase/components/EntityMenu/EntityMenu.jsx

```diff
@@ -1,18 +1,11 @@
 /* eslint-disable react/prop-types */
 import { createRef, Component } from "react";
-import { Motion, spring } from "react-motion";
 import cx from "classnames";
 
-import { isReducedMotionPreferred } from "metabase/lib/dom";
```

Comment:
we don't need this because Mantine obeys prefers-reduced-motion out of the box 🪄